carbon-design-system/carbon-addons-iot-react
# feat(list): load more from top level

Closes #3053

**Summary**
Added feature to let top level items and items without children use the "load more" functionality.

**Change List (commits, features, bugs, etc)**
- Updated List, ListContent and VirtualListContent
- Updated List load more story
- Updated unit tests 

**Acceptance Test (how to verify the PR)**
1. Go to https://deploy-preview-3056--carbon-addons-iot-react.netlify.app?path=/story/1-watson-iot-list--with-load-more
2. Verify that the initially visible "load more" button is working and loads a new row called "Organisation 3", also after and before clicking the other "load more" buttons that are nested under item "Organisation". 
3. Check knob "isVirtualList" and repeat the tests

**Regression Test (how to make sure this PR doesn't break old functionality)**
- The other "load more" buttons work as before



**Things to look for during review**
